ethereum/check_swap_factory_contract_test.go

```diff
@@ -56,8 +62,9 @@ func TestExpectedSwapFactoryBytecodeHex(t *testing.T) {
 // forwarderAddressIndexes slice of trusted forwarder locations is not updated. Use this
 // test to update the slice.
 func TestForwarderAddressIndexes(t *testing.T) {
-	// arbitrary sentinel address that we search for in the contract byte code
-	trustedForwarder := ethcommon.HexToAddress("0x64e902cD8A29bBAefb9D4e2e3A24d8250C606ee7")
```

Comment:
`DeploySwapFactoryWithKey` is registering the forwarder, so the tests are now deploying forwarders instead of using fake addresses.